compiler/rustc_codegen_llvm/src/back/archive.rs

```diff
@@ -189,6 +189,13 @@ impl ArchiveBuilderBuilder for LlvmArchiveBuilderBuilder {
                 path.push(lib_name);
                 path
             };
+            let dlltool_target_arch = match sess.target.arch.as_ref() {
+                "x86_64" => "i386:x86-64",
+                "x86" => "i386",
+                "aarch64" => "arm64",
+                "arm" => "arm",
```

Comment:
Is there any documentation for these labels?

Reply:
Not really, I added a pointer to `llvm-dlltool` where it lists them.